### ISSUE TITLE
Unable to backup /mnt/stateful_partition/crouton/chroots/xenial

### ISSUE THREAD

**Issue by jvschiavo** [dnschneid/crouton#4273]
```
name: xenial
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/xenial...
crouton: version 1-20200428114634~master:67dc8d77
release: xenial
architecture: amd64
xmethod: xorg
targets: xorg,xiwi,xfce,keyboard,cli-extra
host: version 12871.91.0 (Official Build) stable-channel winky 
kernel: Linux localhost 4.4.211-17203-gbe96242e05a2 #1 SMP PREEMPT Thu Apr 23 01:23:05 PDT 2020 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/xenial...
```

#### Please describe your issue:
I'm trying to backup my chroot but I'm getting some errors that I don't know what they mean:
```
chronos@localhost / $ sudo edit-chroot -b xenial
tar: xenial/usr/share/fonts/opentype/noto/NotoSerifCJKsc-Bold.otf: File shrank by 5153556 bytes; padding with zerosr.gz...
tar: xenial/usr/share/fonts/opentype/noto/NotoSansCJKsc-Thin.otf: File shrank by 516676 bytes; padding with zeros
tar: xenial/usr/share/fonts/opentype/noto/NotoSansCJKjp-Thin.otf: File shrank by 12876916 bytes; padding with zeros
tar: xenial/home/joao/.gnupg/S.gpg-agent: socket ignored
Unable to backup /mnt/stateful_partition/crouton/chroots/xenial.
Deleting partial archive.
```

Screenshot:
![Screenshot 2020-05-05 at 15 50 31](https://user-images.githubusercontent.com/20977075/81111255-252b1900-8ef3-11ea-9dd4-cc993e8950cb.png)

Does anyone have any idea? :(

I'd like to backup this chroot before trying to upgrade it to a newer Ubuntu version, since kisak-mesa ppa requires at least 18.04 (but it never got marked as supported on crouton... :/)

**Comment by jvschiavo** [comment]
I thought it could be something storage-related since I had only 2.4GB left on my chromebook internal storage, so I tried again on the SD Card with 6.3GB storage left and got the same results:
![image](https://user-images.githubusercontent.com/20977075/81117635-cdde7600-8efd-11ea-8f4b-106527ecf5cb.png)

:(

**Comment by akabaka74** [comment]
can you show a df -h inside crouton?

**Comment by jvschiavo** [comment]
> can you show a df -h inside crouton?

![Screenshot 2020-05-06 at 21 35 16](https://user-images.githubusercontent.com/20977075/81254160-b0460500-9000-11ea-83df-9e9fbfd1648c.png)

> to backup to a sd card you need to use
> `sudo edit-chroot -f /path/to/folder -b name_of_chroot`

I thought I could just cd into the desired path and it would backup there, but I did that either way and got the same results:
![Screenshot 2020-05-07 at 01 17 07](https://user-images.githubusercontent.com/20977075/81254225-dec3e000-9000-11ea-8ce0-04c19fb2876f.png)

As a reference, my most recent backup is a xenial tar.gz with 2.5 GiB.

Any ideas? :/

**Comment by DennisLfromGA** [comment]
Hmmm, I don't know why `edit-chroot` is bailing out, there doesn't seem to be any error message other than 'Unbale to backup ...'

I have a script I use that incorporates the same method as `edit-chroot` but without entering the chroot. Maybe it'll work for you, might be worth a try. It's called '[chrootbu](https://pastebin.com/hn6dZPek)'. It's not very polished and you may have to look at the script itself to figure out all the command-line options but it works well for me.

*Hope this helps,*
-**DennisLfromGA**

**Comment by jvschiavo** [comment]
@DennisLfromGA Does it create a final tar.gz file just like edit-chroot? That I could restore with edit-chroot too?

I'm also familiar only with the basics of bash, your code looks well written, but I'm not sure how to use it, dealing with variables and that stuff. Could you give me some help? I would appreciate it a lot.

**Comment by jvschiavo** [comment]
Actually, I think I don't need it anymore. I'll update this post soon with more information.

**Comment by Carl393** [comment]
Any new info?

**Comment by jvschiavo** [comment]
> Any new info?

Yes, actually I managed to get around this problem, I took some prints but got lazy to come back and write about it.

Basically, since tar was reporting to have problems trying to backup the directory `usr/share/fonts/opentype` I just deleted it completely, tried again and it worked.

I think my storage got corrupted there since I was getting Input/Output error. I even tried to backup that directory first using rsync and it also had trouble:

![Screenshot 2020-05-08 at 01 45 42](https://user-images.githubusercontent.com/20977075/82211484-49212e00-98e7-11ea-9829-86e9d61caafd.png)

![Screenshot 2020-05-08 at 01 46 33](https://user-images.githubusercontent.com/20977075/82211495-4e7e7880-98e7-11ea-9560-f7b2b7793abf.png)

I have no idea why that happened, it was something really weird, because only some specific fonts seemed to be the problem.

But then I just removed the dir and it worked (with a small error message that I don't know what means):

![Screenshot 2020-05-08 at 01 47 27](https://user-images.githubusercontent.com/20977075/82211519-5b9b6780-98e7-11ea-872d-367deec7b0ef.png)

I had previous backups of `opentype` on other tarballs of my chroots, so I could just copy from them, and it worked.

@Carl393 are you also having problems with it?
